src/styles/global.css

```
@@ -0,0 +1,132 @@
+:root{
+  --bg:#08080c;
+  --bg2:#0d0d14;
+  --card:rgba(255,255,255,.035);
+  --line:rgba(255,255,255,.09);
+  --txt:#f2f2f5;
+  --muted:#9a9aa7;
+  --v:#7c3aed;   /* violet */
+  --p:#ec4899;   /* pink   */
+  --c:#06b6d4;   /* cyan   */
+  --display:"Space Grotesk","Helvetica Neue",Arial,sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)